internal/unpackinfo/lchtimes.go

```diff
@@ -0,0 +1,17 @@
+//go:build darwin || linux
+// +build darwin linux
+
+package unpackinfo
+
+import (
+	"golang.org/x/sys/unix"
+)
+
+// Lchtimes modifies the access and modified timestamps on a target path
+// This capability is only available on Linux and Darwin as of now.
+func (i UnpackInfo) Lchtimes() error {
+	return unix.Lutimes(i.Path, []unix.Timeval{
+		{Sec: i.OriginalAccessTime.Unix()},
+		{Sec: i.OriginalModTime.Unix()}},
+	)
```

Comment:
Both `unix.Timeval` and `time.Time` support more precision than Seconds, is there a particular reason we truncate here?

I've left some comments on the other files about time truncation that stem from this decision.

Reply:
Convo in the other thread about time.